packages/authenticator/amplify_authenticator/example/lib/resolvers/localized_country_resolver.dart

```diff
@@ -6,7 +6,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 /// Our custom country string resolver which calls into our localizations class.
-class LocalizedCountryResolver extends CountryResolver {
+class LocalizedCountryResolver extends DialCodeResolver {
```

Comment:
Since there may be people using this, to avoid a breaking change we need to introduce a forwarding declaration like so (next to the resolver type definition):

```dart
@Deprecated('Use DialCodeResolver instead')
typedef CountryResolver = DialCodeResolver;
```